library/ziti.c

```diff
@@ -170,6 +170,10 @@ int load_tls(ziti_config *cfg, tls_context **ctx) {
     return ZITI_OK;
 }
 
+void ziti_set_client_cert(ziti_context ztx, const char *cert_buf, size_t cert_len, const char *key_buf, size_t key_len) {
+    ztx->tlsCtx->api->set_own_cert(ztx->tlsCtx, cert_buf, cert_len, key_buf, key_len);
```

Comment:
TLS API `set_own_cert` may return an error if key/cert in invalid format or don't match

Reply:
Sure thing, will route an error out and log some context here.

Reply:
I just return the result of the internal call. I am not sure what all the non-zero error return meanings are.